tests/functional/adapter/store_test_failures_tests/test_store_test_failures.py

```diff
@@ -1,9 +1,22 @@
+import pytest
 from dbt.tests.adapter.store_test_failures_tests.test_store_test_failures import (
     StoreTestFailuresBase,
 )
 
 
+TEST_AUDIT_SCHEMA_SUFFIX = "dbt_test__aud"
+
+
 class TestBigQueryStoreTestFailures(StoreTestFailuresBase):
-    def test_store_and_assert(self, project):
+    @pytest.fixture(scope="function")
+    def clean_up(self, project):
```

Comment:
This looks like a teardown method. In my experience, such a method tends to get called `teardown_method` ("method" not because this is a method, but because the scope is "function") and runs automatically.

With that in mind, would you mind if we rename it, add `autouse=True` in the fixture decorator, and remove `clean_up` from the signature of `test_store_and_assert`? It would all behave the same (unless `StoreTestFailuresBase` has additional tests), but it would follow conventions that I've seen.